grails-app/i18n/messages.properties

```diff
@@ -357,7 +357,9 @@ default.credits.label=Credits
 default.balance.label=Balance
 default.dashboard.label=Dashboard
 default.data.label=Data
-default.date.format=dd/MMM/yyyy hh:mm:ss a z
+datetime.format=dd/MMM/yyyy hh:mm:ss a z
+date.format=MM/dd/yyyy
```

Comment:
This should stay as a `default.date.format` due to order of precedence resolving date formats